src/backend/models.py

```diff
@@ -22,17 +22,17 @@ class Check(db.Model):
     name = db.Column(db.String(255), unique=False)
     score = db.Column(db.Integer, unique=False)
     reason = db.Column(db.String(255), unique=False)
-    # TODO details
+    details = db.Column(db.String(255), unique=False)
```

Comment:
The details that scorecard return is a list, which could contain a lot of strings. Storing the entire list as one list is fine in my opinion, but it could get very long, so ```db.Text``` should probably be used here instead of ```db.String```.